### ISSUE TITLE
[BUG] fmDNS: Don't work feature - support proxy-server

### ISSUE THREAD

**Issue by beza2000** [WillyXJ/facileManager#399]
fM Version : **3.0.3**
fmDNS Version : **3.0.4**

- [X] I have read and understood the [contributors guide](https://github.com/WillyXJ/facileManager/blob/master/.github/CONTRIBUTING.md).
- [X] I have checked that the bug-fix I am reporting can be replicated, or that the feature I am suggesting isn't already present.
- [X] I have checked that the issue I'm posting isn't already reported.
- [X] I have checked that the issue I'm posting isn't already solved and no duplicates exist in [closed issues](https://github.com/WillyXJ/facileManager/issues?q=is%3Aissue+is%3Aclosed) and [opened issues](https://github.com/WillyXJ/facileManager/issues)
- [X] I have checked the pull requests tab for existing solutions/implementations to my issue/suggestion.

---

**(BUG | ISSUE) Expected Behavior:**
If I use the option of proxy server, 
fmDNS can download named.root from http://www.internic.net/domain/named.root

**(BUG | ISSUE) Actual Behavior:**
fmDNS doesn't download file and uses named.root from local copy.
From my squid.log:
```
1508324979.759      0 x.x.x.x TAG_NONE/400 3957 GET /domain/named.root - HIER_NONE/- text/html
1508324980.196      0 x.x.x.x TAG_NONE/400 3957 GET /domain/named.root - HIER_NONE/- text/html
```

If I change code and use ` 'request_fulluri' => true,` fmDNS works.
```
diff -Naur facileManagerControl/server/fm-modules/fmDNS/classes/class_buildconf.php facileManager/server/fm-modules/fmDNS/classes/class_buildconf.php
--- facileManagerControl/server/fm-modules/fmDNS/classes/class_buildconf.php    2017-10-11 04:51:06.000000000 +0300
+++ facileManager/server/fm-modules/fmDNS/classes/class_buildconf.php   2017-10-18 14:24:34.774991669 +0300
@@ -1459,6 +1459,7 @@
                if (getOption('proxy_enable')) {
                        $default_opts = array(
                                'http' => array(
+                                       'request_fulluri' => true,
                                        'method' => 'GET',
                                        'proxy' => 'tcp://' . getOption('proxy_host') . ':' . getOption('proxy_port')
                                )
```

**Comment by WillyXJ** [comment]
Would you please submit a pull request with this code change?

**Comment by beza2000** [comment]
I made the pull request - https://github.com/WillyXJ/facileManager/pull/400

**Comment by WillyXJ** [comment]
This is now fixed in [fmDNS 3.1](http://www.facilemanager.com/modules/fmdns/) and later.
